### ISSUE TITLE
Python 3.9?

### ISSUE THREAD

**Issue by dharp** [matterport/Mask_RCNN#2617]
Hello, I'm wondering if you guys plan to upgrade Mask R-CNN to Python 3.9? I get issues with TensorFlow with anything newer than python 3.5.

**Comment by nataliameira** [comment]
@dharp , Try:

pip uninstall keras-nightly
pip uninstall -y tensorflow

pip install h5py==2.10.0
pip install tensorflow==1.15
pip install tensorflow-gpu==1.15.0
pip install keras==2.1.6

**Comment by dharp** [comment]
Hi Natalia,

When I try that with either python 3.8 or 3.9, I get the following error:

ERROR: Could not find a version that satisfies the requirement
tensorflow==1.15 (from versions: 2.5.0rc0, 2.5.0rc1, 2.5.0rc2, 2.5.0rc3,
2.5.0, 2.6.0rc0)
ERROR: No matching distribution found for tensorflow==1.15

**Comment by nataliameira** [comment]
@dharp You must use python 3.6

**Comment by dharp** [comment]
Thanks, Natalia. Any interest on your end in updating Mask R-CNN to newer
python and packages?

Thanks,
Dylan

**Comment by hantongyou** [comment]
I know that you want to make this project in a new environment which is latest. However this project is based on Tensorflow 1.x rather than Tensorflow 2.x. The version of Python,Tensorflow and CUDA is hard to fit. Perhaps you can try this:[link](https://stackoverflow.com/questions/62239835/tensorflow-2-mask-rcnn)
